chrome_extension/mooltipass-content.css

```diff
@@ -904,4 +907,134 @@ input.mp-hover-password {
 
 .mp-genpw-dialog .login-area {
     display: none;
-}
\ No newline at end of file
+}
+
+/* Overlay */
+
+.mp-overlay {
```

Comment:
could you rename mp-overlay to something more explicit like mp-http-auth-overlay ?